includes/class-llms-elementor-migrate.php

```diff
@@ -0,0 +1,273 @@
+<?php
+/**
+ * Handle post migration to the Elementor widgets.
+ *
+ * @package LifterLMS/Classes
+ *
+ * @since [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Handle post migration to the new Elementor widgets.
+ *
+ * @since [version]
+ */
+class LLMS_Elementor_Migrate {
+
+	/**
+	 * Constructor.
+	 *
+	 * @since [version]
+	 */
+	public function __construct() {
+
+		add_action( 'current_screen', array( $this, 'migrate_post' ) );
+		add_action( 'wp', array( $this, 'remove_template_hooks' ) );
+	}
+
+	/**
+	 * Retrieve the elementor data template.
+	 *
+	 * @since [version]
+	 *
+	 * @return array
+	 */
+	public function get_elementor_data_template() {
+		$content = array();
+
+		$content[] = array(
+			'id'       => uniqid(),
+			'elType'   => 'container',
+			'settings' => array(),
+			'elements' => array(
+				array(
+					'id'         => uniqid(),
+					'elType'     => 'widget',
+					'settings'   => array(
+						'content_width' => 'full',
+						'html'          => '<h3>' . esc_attr__( 'Course Information', 'lifterlms' ) . '</h3>',
```

Comment:
There is a heading widget (included in Elementor free) we could use.

Reply:
I did that at first, but the font, color and sizing is all different than the headers added by the lifter course information widgets on the page. By using an HTML widget it inherits from the theme instead. I couldn't find an "inherit" option in the Heading widget but maybe there is one...

Reply:
I'm not seeing a way to ignore color and font when using the Elementor Heading widget, so I think the HTML widget is our best bet to have the styles inherited.

Reply:
Thanks. I figured there was a reason you did it this way. Let's ship it this way for now.